assembly/pom.xml

```diff
@@ -74,6 +74,11 @@
       <artifactId>spark-repl_${scala.binary.version}</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.spark</groupId>
+      <artifactId>spark-connect_${scala.binary.version}</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
May I ask some questions, @grundprinzip and @HyukjinKwon ?
1. Do we need to embed this in the Apache Spark binary releases inevitably?
2. Can we publish like `Kafka` or `Avro`?
3. Can we put this under profile like `hadoop-cloud` at least?

Reply:
This one, I will handle separately. It will be published like Kafka or Avro, and yes it will be covered by a profile. I will do separately.

Reply:
Run `dev/make-distribution.sh --tgz` and decompress the generated tar ball, I found the jars related to `grpc`  and `-rw-r--r--  1 yangjie01  staff    73386  9 27 18:03 protobuf-java-util-3.19.2.jar` are also placed in the jars directory as follows:

```
ls -l *grpc* 
-rw-r--r--  1 yangjie01  staff   256991  9 27 18:03 grpc-api-1.47.0.jar
-rw-r--r--  1 yangjie01  staff    30593  9 27 18:03 grpc-context-1.47.0.jar
-rw-r--r--  1 yangjie01  staff   689433  9 27 18:03 grpc-core-1.47.0.jar
-rw-r--r--  1 yangjie01  staff  9129585  9 27 18:03 grpc-netty-shaded-1.47.0.jar
-rw-r--r--  1 yangjie01  staff     5115  9 27 18:03 grpc-protobuf-1.47.0.jar
-rw-r--r--  1 yangjie01  staff     7570  9 27 18:03 grpc-protobuf-lite-1.47.0.jar
-rw-r--r--  1 yangjie01  staff   838576  9 27 18:03 grpc-services-1.47.0.jar
-rw-r--r--  1 yangjie01  staff    50879  9 27 18:03 grpc-stub-1.47.0.jar
```

should we explicitly exclude them here?

Reply:
same concerns at https://github.com/apache/spark/pull/37710#discussion_r977631501

Reply:
Thanks for pointing this out. I am working on this - would make a PR soon.